packages/rxtranscribe/src/operators/index.js

```diff
@@ -1 +1,3 @@
+export { default as toAWS } from './toAWS';
+export { default as toDeepSpeech } from './toDeepSpeech';
 export { default as transcribe } from './transcribe';
```

Comment:
My plan is to just make an RxJS operator for each speech-to-text strategy.  Then they could be used interchangeably like this:

```javascript
import {share} from 'rxjs/operators';
import {toAWS, toDeepSpeech, toGCP, toDeepgram} from '@bottlenose/rxtranscribe';

const audioWavChunk$ = inputAudioChunk$.pipe(share()); // feed the Observable into a Subject
const awsWord$ = audioWavChunk$.pipe(
  toAWS({region: 'us-east-1})
);
const deepspeechWord$ = audioWavChunk$.pipe(
  toDeepSpeech({modelDir: `${process.env.HOME}/Downloads/deepspeech-0.6.1-models`})
);
const gcpWord$ = audioWavChunk$.pipe(toGCP({foo: 'bar'}));
const deepgramWord$ = audioWavChunk.pipe(toDeepgram());
const word$ = zip(awsTranscript$, deepspeechTranscript$, gcpWord$, deepgramWord$);
word$.subscribe(console.log);
```